tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -840,7 +846,7 @@ trait WebActionsApiBaseTests extends ControllerTestCommon with BeforeAndAfterEac
         }
     }
 
-    it should s"handle http web action with base64 encoded JSON response (auth? ${creds.isDefined})" in {
+    it should s"handle http web action with JSON object as string response (auth? ${creds.isDefined})" in {
```

Comment:
Should this test emulate what `invoke web action ensuring JSON value body arguments are received as is ` does? Meaning running thse as the output ` Seq("this is a string".toJson, JsArray(1.toJson, "str str".toJson, false.toJson), true.toJson, 99.toJson)`?

Reply:
Not necessary since all of these would be returned as string. So no extra coverage gained.

Reply:
Hmm, I don't understand why they would all be a string?

Reply:
in the test, the action is returning JSON values as string.

Reply:
` "body" -> JsObject("field" -> "value".toJson).compactPrint.toJson` the type of `body` is JsString regardless of the actual JSON value.

Reply:
Make sense!